datamodel/APC.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "APC",
+  "description": "Article Processing Charge status",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier for the vocabulary",
+      "default": 1,
```

Comment:
I can see now how the `value` fields are being used. I would suggest that we reconsider this approach - embedding values like this in JSON Schema which are then copied into the JSON payload that's sent to the API is not what JSON Schema is designed for.

In addition to this, the files under `examples/...` all include the `value` attribute. If we choose to keep the `value` attribute in the JSON Schema documents as it is now, I would expect these example JSON files to not include the `value` attribute.